README.md

```diff
@@ -1,6 +1,6 @@
 ![Elixir](https://github.com/elixir-lang/elixir-lang.github.com/raw/master/images/logo/logo.png)
 =========
-[![Build status](https://api.cirrus-ci.com/github/elixir-lang/elixir.svg?branch=master)](https://cirrus-ci.com/github/elixir-lang/elixir)
+![CI](https://github.com/elixir-lang/elixir/workflows/CI/badge.svg) [![Build status](https://api.cirrus-ci.com/github/elixir-lang/elixir.svg?branch=master)](https://cirrus-ci.com/github/elixir-lang/elixir)
```

Comment:
Wouldn't it be nice if it linked to the workflow runs on master when clicked? :)
```suggestion
[![CI](https://github.com/elixir-lang/elixir/workflows/CI/badge.svg)](https://github.com/elixir-lang/elixir/actions?query=branch%3Amaster+workflow%3ACI) [![Build status](https://api.cirrus-ci.com/github/elixir-lang/elixir.svg?branch=master)](https://cirrus-ci.com/github/elixir-lang/elixir)
```

Reply:
Also, to show the badge status image for builds from master only, together with making it clickable:
```suggestion
[![CI](https://github.com/elixir-lang/elixir/workflows/CI/badge.svg?branch=master)](https://github.com/elixir-lang/elixir/actions?query=branch%3Amaster+workflow%3ACI) [![Build status](https://api.cirrus-ci.com/github/elixir-lang/elixir.svg?branch=master)](https://cirrus-ci.com/github/elixir-lang/elixir)
```

Reply:
The linked page shows both CI and CI email and I don't know how to filter by CI only unfortunately - this may be a quirk of this particular GitHub search field

Reply:
Applied. Thanks!

Reply:
I am guessing it should be possible to show only the CI stuff with a workaround this GitHub quirk if we renamed the CI workflow. For example searching for `CI Linux` shouldn't return `CI email` results, I think